crowbar_framework/lib/crowbar/repository.rb

```diff
@@ -16,16 +16,50 @@
 
 module Crowbar
   class Repository
-    attr_reader :platform, :id, :registry
+    attr_reader :platform, :id, :config
 
     class << self
       def load!
-        @config = YAML.load_file(Rails.root.join("config/repos.yml"))
+        etc_yml = "/etc/crowbar/repos.yml"
+
+        if Crowbar::Product.is_ses?
+          @all_repos = YAML.load_file(Rails.root.join("config/repos-ses.yml"))
+        else
+          @all_repos = YAML.load_file(Rails.root.join("config/repos-cloud.yml"))
+        end
+
+        # merge data from etc config file
+        etc_repos = {}
+        if File.exist? etc_yml
+          begin
+            etc_repos = YAML.load_file(etc_yml)
+          rescue SyntaxError
+            # ok, let's live without it
```

Comment:
i would put at least a log message here. wouldn't it run it that rescue all over again without anyone noticing?